src/protocol/definitions/ping360.json

```diff
@@ -28,34 +28,36 @@
                         "description": "Operating mode (1 for Ping360)"
                     },
                     {
-                        "name": "gain_index",
+                        "name": "gain_setting",
                         "type": "u8",
-                        "description": "Analog gain index (0 = low, 1 = normal, 2 = high)"
+                        "description": "Analog gain setting (0 = low, 1 = normal, 2 = high)"
                     },
                     {
                         "name": "angle",
                         "type": "u16",
-                        "description": "Head angle in gradians (3 ASCII/Hex digits 0 to 399 decimal = 0 to 360 degrees)"
+                        "description": "Head angle",
+                        "units": "gradian"
                     },
                     {
-                        "name": "pulse_usec",
+                        "name": "transmit_duration",
```

Comment:
In Ping1D we use `pulse_usec`, what is the difference between this variable and the one used in Ping1D ? From the description it appears to be the same thing.

Reply:
I think it is the same thing. Transmit duration is descriptive. pulse_usec is not. We may change ping1d also, but you know then the code and doc changes too..

Reply:
Well, so we should update Ping1D to use the same names if possible. Since we are going to release Ping360 probably with a major release version, we can/should break binary compatibility of the API for Ping1D.

Reply:
https://github.com/bluerobotics/ping-protocol/issues/152